src/custom/components/swap/TradePrice/TradePriceMod.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
```

Comment:
Basically restored to previous version prior to David's mods.

Reply:
can just delete the mod as it's no longer used

Reply:
actually do delete the mod please